cacher/cacher.go

```diff
@@ -134,6 +136,18 @@ func (cacher *Cacher) dispatch(id [32]byte, msg http.RequestWithResponder) {
 				if tx.Tx.Output.String() == pack.NewTyped().String() {
 					return true
 				}
+
+				var output engine.LockMintBurnReleaseOutput
+				err = pack.Decode(&output, tx.Tx.Output)
+				if err != nil {
+					// it might a different kind of output, so it's ok
+				} else {
+					v1Tx, err := v1.QueryTxFromV2QueryTx(tx)
```

Comment:
Can we get away with just converting the output? Or do we need to convert the entire transaction?